langlib/lang.typedesc/src/main/ballerina/typedesc_lib.bal

```diff
@@ -40,6 +40,22 @@ public type TypeId readonly & record {|
 |};
 
 # Returns the type-ids induced by a typedesc value.
+# 
+# ```ballerina
+# type Error distinct error;
+# 
+# type SampleError distinct (Error & error<record {string msg;}>);
+# 
+# SampleError.typeIds() ⇒ [
+#       {"moduleId":{"organization":"$anon","name":".","platformParts":["0"]},"localId":"SampleError"},
+#       {"moduleId":{"organization":"$anon","name":".","platformParts":["0"]},"localId":"Error"}
+# ]
```

Comment:
What is this output? Please make sure it is the same as what is printed when we use `io:println`, or once we write tests they will fail.

Reply:
It was formatted previously considering the char limit. But fixed it considering the expected verification tests in https://github.com/ballerina-platform/ballerina-lang/pull/39111/commits/3779031218f05300eec7b586599bc5a5269743c1